addons/main/functions/fn_clearTable.sqf

```diff
@@ -18,5 +18,8 @@
 
 params ["_table"];
 private _tableObjects = _table getVariable ["sebs_briefing_table_tableObjects", []];
-{deleteVehicle _x} forEach _tableObjects;
-_table setVariable ["sebs_briefing_table_tableObjects", []];
\ No newline at end of file
+{
+deleteVehicle _x;
+sleep 0.4; //may need [] spawn { for it 
```

Comment:
Too long. I wonder if there is a way of doing single frame sleeps? idk.

Reply:
Sleep 0.001; should suspend until the next frame.

Reply:
you could use [onEachFrame](https://community.bistudio.com/wiki/onEachFrame)

Reply:
> you could use [onEachFrame](https://community.bistudio.com/wiki/onEachFrame)

as a replacement to "forEach"

Reply:
```suggestion
{deleteVehicle _x} forEach _tableObjects;
_table setVariable ["sebs_briefing_table_tableObjects", []]; 
{
deleteVehicle _x;
```
onEachFrame { 
deleteVehicle _table[0];
_table deleteAt 0;
};

Or something like that

Reply:
Test it and make some more commits. Currently I'm of the opinion using a scheduled env will cause more problems than it solves, but I'm happy to be wrong. I suggest using the example mission as a base, as you don't have to build the addon using that.